CHANGES.rst

```diff
@@ -6,6 +6,9 @@ There's a frood who really knows where his towel is.
 2.10.1 (unreleased)
 ^^^^^^^^^^^^^^^^^^^
 
+- Update plugins javascript (closes `#117 <https://github.com/collective/sc.social.like/issues/117>`_).
```

Comment:
this is not closing the issue; the issue serves only as a reference; the changelog entry must be something like: **Facebook's widget code was updated… blah, blah, blah.**

we'll take care of that later.